### PR TITLE
[docs-only] add monitoring and tracing for ocis_traefik

### PR DESCRIPTION
**Pull request by wkloucek**
Add documentation on how our monitoring and tracing story / infrastructure looks like. Add optional monitoring and tracing configuration for our deployment examples for a simple start.

External documentation in https://github.com/owncloud-devops/monitoring-tracing-server and https://github.com/owncloud-devops/monitoring-tracing-client is currently worked on.

Screenshot of the new page in docs:
image

### CODE REVIEW

File: deployments/examples/cs3_users_ocis/.env [hunk 1]
```diff
@@ -24,3 +24,9 @@ LDAP_ADMIN_PASSWORD=
 ### LDAP manager settings ###
 # Domain of LDAP manager. Defaults to "ldap.owncloud.test"
 LDAP_MANAGER_DOMAIN=
+
+
+# If you want to use debugging and tracing with this stack:
+# you need to enable
```
Comment (butonic) [inline deployments/examples/cs3_users_ocis/.env:30]: enable what? reads like something is missing...

File: deployments/examples/ocis_keycloak/.env [hunk 2]
```diff
@@ -27,3 +27,9 @@ KEYCLOAK_REALM=
 KEYCLOAK_ADMIN_USER=
 # Admin user login password. Defaults to "admin"
 KEYCLOAK_ADMIN_PASSWORD=
+
+
+# If you want to use debugging and tracing with this stack:
+# you need to enable
```
Comment (butonic) [inline deployments/examples/ocis_keycloak/.env:33]: same

File: deployments/examples/ocis_traefik/.env [hunk 3]
```diff
@@ -15,3 +15,9 @@ TRAEFIK_ACME_MAIL=
 OCIS_DOCKER_TAG=
 # Domain of oCIS, where you can find the frontend. Defaults to "ocis.owncloud.test"
 OCIS_DOMAIN=
+
+
+# If you want to use debugging and tracing with this stack:
+# you need to enable
```
Comment (butonic) [inline deployments/examples/ocis_traefik/.env:21]: same

File: deployments/examples/owncloud10_with_oc_web/.env [hunk 4]
```diff
@@ -19,3 +19,9 @@ OCIS_DOMAIN=
 ### oC10 ###
 # Domain of ownCloud 10, where you can find the frontend. Defaults to "oc10.owncloud.test"
 #OC10_DOMAIN=
+
+
+# If you want to use debugging and tracing with this stack:
+# you need to enable
```
Comment (butonic) [inline deployments/examples/owncloud10_with_oc_web/.env:25]: same

**Review by butonic — CHANGES_REQUESTED** [review]

**Review by micbar — APPROVED** [review]
LGTM

**Pull request merged**